lib/recma-jsx-build.js

```diff
@@ -41,6 +37,7 @@ export function recmaJsxBuild({contain}) {
                 u('Property', {
                   kind: 'init',
                   shorthand: false,
+                  // @ts-ignore assume specifier is not a default export
```

Comment:
This is something to note, I'm not sure if it's a quirk of the typing or potentially an edge case not covered

Reply:
`build` does not generate a default export, to in this case it’s fine!